parts/linux/cloud-init/artifacts/configure_azure0.sh

```diff
@@ -1,10 +1,18 @@
 #!/usr/bin/env bash
 
+# It's only necessary to configure azure0 in Ubuntu 18.04
+lsb_release -ir | grep 18.04 &> /dev/null
```

Comment:
we may have non ubuntu os in future.
how about this:

lsb_release -i | awk -F':' '{print $2}'| awk '{$1=$1; print $1}' | grep -qi "^ubuntu$"
if [ $? != 0 ]; then
    echo 'It is not Ubuntu  Skip configuring azure0'
    exit 0
fi

lsb_release -r | awk -F':' '{print $2}'| awk '{$1=$1; print $1}' | grep -q "^18.04$"
if [ $? != 0 ]; then
    echo 'It is not Ubuntu  18.04. Skip configuring azure0'
    exit 0
fi

Reply:
Thanks for the suggestion.